notebooks/mos_spectroscopy_advanced/MOSspec_advanced.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>The cell <strong>input the appropriate spectrum and region </strong>part of the workflow of the API calls is currently dependent on the previous section to run successfully. Can we explore ways to make this section more self-contained?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Subset 2 should be generated by the API call above, but not 'spec1d masked'. I am adding a couple of cells above to make this more robust.</p>

Reply:
Thank you!